tracing-appender/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 0.2.3 (November 6, 2023)
+
+This release contains several new features.
+
+### Added
+
+- **rolling**: add option to automatically delete old log files ([#2323])
+- **non_blocking**: allow worker thread name to be configured ([#2365])
+- **rolling**: add a builder for constructing `RollingFileAppender`s ([#2227])
+- **rolling**: add `Builder::filename_suffix` parameter ([#2225])
+
+### Changed
+
+- **non_blocking**: name spawned threads ([#2219])
```

Comment:
this feels like "added" to me, but your mileage may vary :woman_shrugging:

Reply:
I suppose it's `added`. I'll make "changed" stuff for breaking changes and otherwise internal, but noteworthy, fixes.